pkg/helpers/helpers.go

```diff
@@ -663,3 +664,34 @@ func SyncSecret(client, targetClient coreclientv1.SecretsGetter, recorder events
 		return resourceapply.ApplySecret(targetClient, recorder, source)
 	}
 }
+
+func GetHubKubeconfig(ctx context.Context,
```

Comment:
this name is confusing, and please add more comment on this

Reply:
Yes, comment added and also change the argument from `kubeconfig` to `managementKubeconfig`.